README.md

```diff
@@ -97,37 +102,41 @@ With Finetuner, one can easily uplift pre-trained models to be more performant a
 </tbody>
 </table>
 
-<sub><sup>All metrics are evaluated on k@20 after training for 5 epochs using Adam optimizer with learning rates of 1e-4 for ResNet, 1e-7 for CLIP and 1e-5 for the BERT models.</sup></sub>
+<span style="font-size:small">All metrics were evaluated for k@20 after training for 5 epochs using the Adam optimizer with learning rates of 1e-4 for ResNet, 1e-7 for CLIP and 1e-5 for the BERT models.</span>
```

Comment:
can you only keep the edit change, not change the style? this is aligned with jina and docarray

Reply:
Yes, but I only made the change because @gmastrapas complained about the text size. If this is the company style, I'll leave it.